### PR TITLE
react-native-config and react-native-linear-gradient support

### PR DESCRIPTION
**Pull request by maxhora**
- Support of react-native-config
- Basic support of react-native-linear-gradient (rendered as empty View so far).
Used version is 2.3.0, since the latest 2.4.0 has changes to support compatibility with RN v0.51
- JSC_ENABLED from config is set to 0
- STUB_STATUS_GO from config is set to 1 to avoid referencing of not yet implemented react-native-status module with desktop platform support

### CODE REVIEW

File: src/status_im/desktop/core.cljs [hunk 1]
```diff
@@ -41,4 +42,5 @@
 (defn init []
   (enable-console-print!)
   (check-3rdparty-libraries)
+  (print "value of log-level from config: "config/log-level)
```
Comment (vkjr) [inline src/status_im/desktop/core.cljs]: What do you think about moving this to `check-3rdparty-libraries` functions?

File: package.json [hunk 2]
```diff
@@ -70,7 +72,7 @@
     "react-native-image-resizer": "1.0.0",
     "react-native-invertible-scroll-view": "1.1.0",
     "react-native-level-fs": "^3.0.0",
-    "react-native-linear-gradient": "2.3.0",
+    "react-native-linear-gradient": "git+https://github.com/MaxRis/react-native-linear-gradient.git#desktop-platform-v230",
```
Comment (vkjr) [inline package.json]: Probably we need also to create fork in `status-im` repository with `react-native-linear-gradient`.

File: src/status_im/ui/screens/views.cljs [hunk 3]
```diff
@@ -23,6 +28,9 @@
         (let [component (case current-view
                           :login login
                           :profile profile
+                          :new-chat new-chat
```
Comment (vkjr) [inline src/status_im/ui/screens/views.cljs:31]: Can we already render these screens (chat, new-chat, new-public-chat)?

**Review by vkjr — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/views.cljs (hunk 3 above)
Comment (maxhora) [reply src/status_im/ui/screens/views.cljs:31]: @vkjr not yet, but I'm looking what is failing for the "chat" screen

**Comment by maxhora** [comment]
@vkjr thank you! Have fixed things you mentioned

**Pull request merged**